src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GroupFeedback/hooks.js

```diff
@@ -7,14 +7,14 @@ export const state = {
   summary: (val) => useState(val),
 };
 
-export const groupFeedbackCardHooks = (groupFeedbacks, updateSettings) => {
+export const groupFeedbackCardHooks = (groupFeedbacks, updateSettings, answers) => {
   const [summary, setSummary] = module.state.summary({ message: messages.noGroupFeedbackSummary, values: {} });
 
   useEffect(() => {
     if (groupFeedbacks.length === 0) {
       setSummary({ message: messages.noGroupFeedbackSummary, values: {} });
     } else {
-      const feedbacksInList = groupFeedbacks.map(({ answers, feedback }) => `${answers.toString()} ${feedback}\n`);
+      const feedbacksInList = groupFeedbacks.map(({ answerslist, feedback }) => `${answerslist.filter((a) => answers.includes(a)).toString} ${feedback}\n`);
```

Comment:
I'm getting this error when hitting the "add group feedback" button
<img width="785" alt="image" src="https://user-images.githubusercontent.com/56318341/217925068-725a7953-0226-4ea2-bcbe-04318122197d.png">

Reply:
yeah -- fix incoming

Reply:
turns out I just needed it to have another useeffect dependancy